.travis.yml

```diff
@@ -95,14 +72,15 @@ matrix:
 
     - name: cargo clippy
       rust: nightly
-      script:
-        - if ! rustup component add clippy; then
+      install:
+        - if ! rustup component add clippy 2>/dev/null; then
```

Comment:
Alternatively the `cargo clippy` build could be moved to use `stable` now, there's no reason this needs nightly afaik?

Reply:
Some unstable features require nightly rustc:
https://github.com/rust-lang-nursery/futures-rs/blob/d309ea397581efefeacaa5a928d33b0ec6e87834/futures-core/src/lib.rs#L3
https://github.com/rust-lang-nursery/futures-rs/blob/d309ea397581efefeacaa5a928d33b0ec6e87834/futures-io/src/lib.rs#L11